### ISSUE TITLE
[ENH] Create a support_ attribute to avoid error in generating a dataframe report or to select in a sklearn way

### ISSUE THREAD

**Issue by cfrancois7** [Ekeany/Boruta-Shap#38]
Current Situation
=================
Currently, the selector has mutliple **columns** attribute.
Also, it not easy to apply the selection of features through an attribute "support_" similar to sklearn framework.
Also, it is not sure, except if we go into the code, that the order of hits is similar to the order of columns of the dataframe provided as argument.

Enhancement
===========
- Create a "support_" True/False mask such as "support_" attribute from 'sklearn.feature_selection' object.
- Provide an example or add in the documentation the means to create a "report" dataframe of hits and "accepted", such as f_classif test : ([list of hits values], [True/False list of accepted features]) the whole ordered as the X provided as argument.

**Comment by Ekeany** [comment]
Hi,

If I understand correctly you would like to be able to view the number of hits received for each feature ?. You can certainly make a pull request if you want. 

The hits are stored in self.hits and then the column names are stored as self.all_columns so it is just a matter of creating a function to merge these to into a dataframe and then you will need to map the self.rejected_columns = [] self.accepted_columns = [] lists to create a column of which features were accepted.
